src/common/helper.js

```diff
@@ -1392,6 +1392,34 @@ async function getSkillById (skillId) {
   return _.pick(res.body, ['id', 'name'])
 }
 
+/**
+ * Function to get skills by names
+ * 
+ * @param {Array<String>} names - the skill name
+ * @returns the request result
+ */
+async function getSkillsByExactNames (names) {
+  if (!names || !_.isArray(names) || !names.length) {
+    return []
+  }
+  const token = await getM2MToken()
```

Comment:
Adding spaces in this function would make the code blocks more readable, especially since single function calls in JS can span multiple lines with multiple levels of indentation, dot notations, and closing `)` and `}` on different lines. For this function, I would add a blank line at the following lines, where the intent of the code changes:
- 1405
- 1406
- 1410 - we define the `url` that's used in the `request`, so 1410-1415 stay together
- 1416
- 1420

Please consider this.

Reply:
Fixed